sysid/src/main/native/include/sysid/view/DataSelector.h

```diff
@@ -74,8 +75,10 @@ class DataSelector : public glass::View {
   int m_selectedAnalysis = 0;
   std::future<TestData> m_testdataFuture;
   std::vector<std::string> m_testdataStats;
+  std::vector<std::string> kValidTests;
```

Comment:
This isn't initialized with the test names. The k prefix implies it's a constant.